tests/zfs-tests/include/commands.cfg.in

```diff
@@ -113,8 +120,10 @@ export UNSHARE="@UNSHARE@"
 export USERADD="@USERADD@"
 export USERDEL="@USERDEL@"
 export USERMOD="@USERMOD@"
+export VMSTAT="@VMSTAT@"
 export WAIT="@WAIT@"
 export WC="@WC@"
+export ZINJECT="@ZINJECT@"
```

Comment:
ZINJECT doesn't need to be added here.  It's not a system tool and is already available from `zfs_commands.cfg`.  Dropping this line should fix the zpool_scrub* failures.